doc/api/magic-sql.md

```diff
@@ -359,3 +359,21 @@ LIMIT 3
 ```{code-cell} ipython3
 %sql --file my-query.sql
 ```
+
+## Parameterizing arguments
+
+JupySQL supports variable expansion of arguments in the form of `{{variable}}`. This allows the user to specify arguments with placeholders that can be replaced by variables dynamically. The arguments supported are `section`, `close`, `alias`, `save` and `with`. 
```

Comment:
Arguments list is still valid. But I've removed this sentence because everytime a new argument is added we need to remember to add to this list: https://github.com/ploomber/jupysql/pull/941/commits/a2c9266287d19c035e3e441580292fb0d72ec647